src/internal/account.ts

```diff
@@ -194,8 +191,10 @@ export async function getAccountTokensCount(args: {
     query: graphqlQuery,
     originMethod: "getAccountTokensCount",
   });
-
-  return data.current_token_ownerships_v2_aggregate.aggregate;
+  if (!data.current_token_ownerships_v2_aggregate.aggregate) {
+    throw Error("Failed to get the count of account tokens");
```

Comment:
I guess I am just confused why we suddenly throw here? it doesn't seem like a pattern we use in other queries... also is it really a failure? because it is not a server error...

Reply:
I don't think it will actually ever practically throw, but the generated type data.current_token_ownerships_v2_aggregate.aggregate is optional so you need a check.  We can revisit later